quinn/src/delay_queue.rs

```diff
@@ -0,0 +1,442 @@
+use std::{fmt, ops::RangeInclusive};
+
+use slab::Slab;
+
+/// Stores values to be yielded at specific times in the future
+#[derive(Debug)]
+pub struct DelayQueue<T> {
+    /// Definitions of each active timer
+    timers: Slab<TimerState<T>>,
+
+    /// Hierarchical timer wheel
+    ///
+    /// Timers are bucketed into slots organized into levels. Level n spans 2^(`LOG_2_SLOTS` *
```

Comment:
are the backticks here for the sake of having a rustdoc link to the constant? For comment readability the might be better placed around the whole math (before `2^`). Not sure if rustdoc actually does something for private fields and private constants.

Reply:
They were just to express that it's a constant, but I was ambivalent. Reformatted as advised.